index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Apple (clon)</title>
+</head>
+
+<body>
+
+  <nav id="main-navbar">
+    <div class="top-bar">
+      <ul class="ul-bar">
+        <li class="btn first-btn"><span class="inner-border"></span></li>
+        <li class="btn"><span class="inner-border">Store</span></li>
+        <li class="btn"><span class="inner-border">Mac</span></li>
+        <li class="btn"><span class="inner-border">iPod</span></li>
+        <li class="btn"><span class="inner-border">iPhone</span></li>
+        <li class="btn"><span class="inner-border">iPad</span></li>
+        <li class="btn"><span class="inner-border">iTunes</span></li>
+        <li class="btn"><span class="inner-border">Support</span></li>
+      </ul>
+      <div id="search-div">
+        <form action="">
+          <span><i class="fas fa-search"></i></span>
+          <input type="text" class="search">
+        </form>
+      </div>
+    </div>
+  </nav>
+
+  <section id="main-section">
+    <div class="body-image-main">
+      <div class="header-text">
+        <div id="verse"></div>
+        <div id="share"></div>
+        <div id="video"></div>
+      </div>
+    </div>
+  </section>
+
+  <section>
+    <div id="bottom-section">
+      <div class="bottom"></div>
+      <div class="bottom"></div>
+      <div class="bottom"></div>
+      <div class="bottom"></div>
+    </div>
+  </section>
+  
+  <footer id="fondo">
+    <div class="shop">
+      <div>Shop the <a href="#">Apple Online Store</a> (1-800-MY-APPLE), visit an <a href="#">Apple Retail Store</a>,
+        or find a <a href="#">reseller</a>.</div>
+      <div id="disclaimers">
+        <span id="copyright">Copyright © 2014 Apple Inc. All rights reserved</span>
+        <span id="terms">
+          <ul>
+            <li><a href="#">Terms of Use</a></li>
+            <li><a href="#">Privacy Policy</a></li>
+          </ul>
+        </span>
+      </div>
+    </div>
+
+    <div id="footer-links">
+      <ul>
+        <li id="uno" class="lista">Site Map</li>
+        <li class="lista">Hot News</li>
+        <li class="lista">RSS Feeds</li>
+        <li class="lista">Media Info</li>
+        <li class="lista">Environment</li>
+        <li class="lista">Job Opportunities</li>
+        <li class="lista">Contact us</li>
+        <li class="lista">
+          <img src="img/globe.png" alt="country">
+        </li>
+      </ul>
+    </div>
+  </footer>
+</body>
+<link rel="stylesheet" href="css/styles.css">
```

Comment:
The `link` tags go inside the `head` tag. https://www.w3schools.com/Tags/tag_link.asp